WordPressCom-Stats-iOS/WPStatsService.m

```diff
@@ -78,7 +78,7 @@ - (void)retrieveAllStatsForDate:(NSDate *)date
     NSMutableDictionary *cacheDictionary = [self.ephemory objectForKey:@[@"BatchStats", @(unit), endDate]];
     DDLogVerbose(@"Cache count: %@", @(cacheDictionary.count));
     
-    if (cacheDictionary && cacheDictionary.count == 13) {
+    if (cacheDictionary && cacheDictionary.count == 15) {
```

Comment:
I'm going to open another issue for this - it's a poor design for the determination if the cache is full for a request.